app/api/common/serializers.py

```
@@ -32,8 +32,22 @@ def _prefer_youtube_hq_thumbnail(url: str | None) -> str | None:
+    if repo is not None and getattr(engine.state, "now_playing_id", None) is not None:
+        try:
+            liked = repo.get_playlist_by_source_url("custom://liked_songs")
+            if liked is not None:
+                item = repo.get_item(engine.state.now_playing_id)
+                if item is not None:
+                    now_playing_is_liked = repo.playlist_contains_track(
+                        liked.id,
+                        normalized_url=getattr(item, "normalized_url", None),
+                        provider_item_id=getattr(item, "provider_item_id", None),
+                    )
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)